src/oidcc_provider_configuration.erl

```diff
@@ -34,7 +34,8 @@
 
 -type quirks() :: #{
     allow_issuer_mismatch => boolean(),
-    allow_unsafe_http => boolean()
+    allow_unsafe_http => boolean(),
+    document_overrides => map()
```

Comment:
Should we type this or is map enough?

Reply:
The keys are definitely binaries, so I can at least do that much. The values could be anything, but in practice they're booleans, binaries, or lists of binaries.

Reply:
Ah sorry, this is applied before we convert to structured information. Ignore this.